src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/types/openapi.d.ts

```diff
@@ -279,11 +286,10 @@ declare namespace Components {
       idp?: string | null;
     }
     export type OneYearRenewalexplanations =
-      | "Icouldnotfindemploymenttocompletetherequiredhours"
-      | "Icouldnotworkduetomyvisastatusstudentvisaexpiredvisa"
-      | "IliveandworkinacommunitywithoutothercertifiedECEs"
-      | "Iwasunabletoenterthecountryasexpected"
-      | "Iwasunabletoworkinthechildcarefieldforpersonalreasons"
+      | "IIleftthechildcarefieldforpersonalreasons"
+      | "Iwasunabletocompletetherequiredhoursofprofessionaldevelopment"
+      | "Iwasunabletofindemploymentinthechildcarefieldinmycommunity"
+      | "MyemploymentdiddoesnotrequirecertificationasanECEforexamplenannyteachercollegeinstructoradministratoretc"
```

Comment:
`MyemploymentdiddoesnotrequirecertificationasanECEforexamplenannyteachercollegeinstructoradministratoretc` that's a mouthful 😉 `diddoes` intentional?

Reply:
haha 100% intentional - Took it straight from Dynamics :P